quinn-h3/src/server.rs

```diff
@@ -366,6 +791,10 @@ impl Sender {
         Ok(BodyWriter::new(send, self.conn, self.stream_id, true))
     }
 
+    /// Terminate the request by rejecting it.
```

Comment:
The method is called `cancel()`, but you describe it as "Terminate (...) rejecting". I'm confused now!

Reply:
Fixed it and updated the docs with a short explanation.